<launch>

  <arg name="namespace" default="/dvrk/"/>
  <arg name="master" default="MTMR"/>   
  <arg name="slave" default="PSM1"/>   
  <arg name="nri_config" default="$(find dvrk_nri_robot)/config/cmu-dVRK/nri-console-$(arg master)-$(arg slave)-Teleop.json"/>
  <arg name="scale" default=".95" />
  <arg name="camera_registration" default="$(find dvrk_vision)/defaults/registration_params_jhu.yaml" />

  <include file="$(find dvrk_vision)/launch/just_cams_jhu.launch" />
  
  <node ns="stereo" name="main_gui" type="user_study_gui.py" pkg="dvrk_vision" output="screen" clear_params="true" required="true">
    <param name="camera_registration" value="$(arg camera_registration)" />
    <param name="marker_topic" value="$(arg namespace)$(arg master)_$(arg slave)/proxy_slave_phantom" />
    <param name="robot_frame" value="$(arg slave)_SIM_psm_base_link"/>
    <param name="end_effector_frame" value="vf_point"/>
    <param name="psm_name" value="$(arg slave)"/>
    <param name="camera_frame" value="stereo_camera_frame"/>
  </node>

</launch>
